sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -36,12 +36,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.11.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.12.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
Why do we need unreleased azure-core dependency for app config?

Reply:
Need to update all usages of `azure-core` and `azure-core-http-netty` to unreleased as those libraries contain compile time dependencies for Reactor and Reactor Netty which we are upgrading across versions which major breaking changes. Using the release version will cause a dependency breakage.